src/ragger/backend/interface.py

```diff
@@ -30,11 +30,15 @@ class RaisePolicy(Enum):
     RAISE_NOTHING = auto()
     RAISE_ALL_BUT_0x9000 = auto()
     RAISE_ALL = auto()
+    RAISE_CUSTOM = auto()
 
 
 class BackendInterface(ABC):
 
-    def __init__(self, firmware: Firmware, log_apdu_file: Optional[Path] = None):
+    def __init__(self,
+                 firmware: Firmware,
+                 log_apdu_file: Optional[Path] = None,
+                 whitelisted_status: Optional[list] = None):
```

Comment:
If the default value is a list, might as well defining it here. It will simplify the following code.
```suggestion
                 whitelisted_status: list = []):
```

Reply:
I did that at first, but I had a codesmell detected :(

Reply:
Probably because of dangerous side-effect, but we don't trigger that pitfall.
Don't know how to bypass though :thinking:

Reply:
Maybe by using tuples instead of list? It stays immutable, that's consistent with our usage.

Reply:
```suggestion
                 whitelisted_status: Iterable = ()):
```

Reply:
It was somehting about setting the default value of a variable in a function being bad practice for maintenability :thinking: